crates/ruma-client-api/src/state/get_state_events_for_key.rs

```diff
@@ -86,8 +86,8 @@ pub mod v3 {
             use percent_encoding::{utf8_percent_encode, NON_ALPHANUMERIC};
 
             let room_id_percent = utf8_percent_encode(self.room_id.as_str(), NON_ALPHANUMERIC);
-            let event_type_percent =
-                utf8_percent_encode(self.event_type.as_str(), NON_ALPHANUMERIC);
+            let event_type = self.event_type.to_string();
```

Comment:
I see `to_cow_str` defined down below, maybe thats better to avoid useless allocations?

ditto for same change below

Reply:
I don't think that's very important since it's just a microoptimization. `to_cow_str` is private and I'm not sure it makes sense to make it public.

Reply:
Alright, just thought you forgot about the function, but in this case this is fine enough then :+1: